src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -613,7 +613,7 @@ internal static List<TargetFrameworkInformation> GetTargetFrameworkInfos(IReadOn
 
                 targetFrameworkInformation.FrameworkReferences.AddRange(GetFrameworkReferences(msBuildProjectInstance));
 
-                if (isCpvmEnabled && targetFrameworkInformation.Dependencies.Any())
+                if (isCpvmEnabled)
```

Comment:
Fixes https://github.com/NuGet/Home/issues/10327

Reply:
I've updated the PR description as well